docker/dockerfile.tri

```
@@ -156,6 +156,8 @@ RUN git clone https://github.com/rapidsai/asvdb.git /repos/asvdb && cd /repos/as
 RUN git clone https://github.com/NVIDIA/NVTabular.git /nvtabular/ && \
     cd /nvtabular/; if [ "$RELEASE" == "true" ] && [ ${NVTAB_VER} != "vnightly" ] ; then git fetch --all --tags && git checkout tags/${NVTAB_VER}; else git checkout main; fi; \
     python setup.py develop --user;
+# Install Transformers4Rec
+RUN pip install git+git://github.com/NVIDIA-Merlin/Transformers4Rec.git#egg=transformers4rec[all]
```

Does this install the TensorFlow python package? I don't think we need that here (since we should be using the triton TF backend for serving up TF models from transformers4rec).

Since we're triton python backend for PyTorch models w/ transformers4rec we will need the python pytorch dependencies.  How about:

```suggestion
RUN pip install git+git://github.com/NVIDIA-Merlin/Transformers4Rec.git#egg=transformers4rec[pytorch,nvtabular]```